tbump.toml

```diff
@@ -1,7 +1,7 @@
 github_url = "https://github.com/PyCQA/pylint"
 
 [version]
-current = "2.16.0-dev"
+current = "2.16.0-b0"
```

Comment:
```suggestion
current = "2.16.0b0"
```
As suggested by the relevant PEP:
https://peps.python.org/pep-0440/#pre-releases

Reply:
Nice catch I was going to release a defective version.

Reply:
I don't think it matters, you can use what you want. But it's nice to follow "ecosystem standards" :smile:

Reply:
Especially since we are a linter that enforces "ecosystem standards" :smile: